src/redux/reducers/backendStatus/backendStatusUpdated.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-param-reassign */
+import _ from 'lodash';
+import produce from 'immer';
+
+import initialState from './initialState';
+
+const backendStatusUpdated = produce((draft, action) => {
+  const { experimentId, status } = action.payload;
+
+  _.merge(draft[experimentId]?.status, status);
```

Comment:
Why are we using `_.merge` here? why not just replace the old `status` with the new one?

Reply:
Because these upadtes are an object like this:
`{ gem2s: {...} }` or `{ pipeline: {...} }`

While the backendStatus stored in redux is like this:
`{ gem2s: {...}, pipeline: {...} }`